cookbooks/cdo-varnish/libraries/helpers.rb

```diff
@@ -76,14 +78,14 @@ def valid_path?(path)
   return false if path.length > 255
   # Valid characters allowed
   char = /[A-Za-z0-9_\-.$\/~"'@:+]/
-  # Exactly one wildcard required at start or end
-  return false unless path.match /^(\*#{char}*|#{char}*\*)$/
+  # Maximum one wildcard allowed at start or end
+  return false unless path.match /^(\*#{char}*|#{char}*\*|#{char}*)$/
```

Comment:
Good suggestions, thanks! The `x` flag can looks very useful, I've never used it before.